arrows/mvg/camera_options.cxx

```diff
@@ -0,0 +1,518 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief Implementation of MVG camera options.
+ */
+
+#include "camera_options.h"
+#include "lens_distortion.h"
+
+using namespace kwiver::vital;
+
+#define MVG_ENUM_HELPERS(NS, mvg_type) \
+namespace kwiver { \
+namespace vital { \
+\
+template<> \
+config_block_value_t \
+config_block_set_value_cast(NS::mvg_type const& value) \
+{\
+  return NS::mvg_type##ToString(value);\
+}\
+\
+template<> \
+NS::mvg_type \
+config_block_get_value_cast(config_block_value_t const& value)\
+{\
+  NS::mvg_type cet;\
+  if(!NS::StringTo##mvg_type(value, &cet)) \
+    VITAL_THROW( bad_config_block_cast,value);\
+  return cet;\
+}\
+\
+}\
+\
+namespace arrows { \
+namespace mvg { \
+\
+template<>                                                              \
+std::string                                                             \
+mvg_options< NS::mvg_type >()                                       \
```

Comment:
I suggest we align the `\` characters for this macro as they were in the original code to make it more obvious that this is one big macro.

Reply:
fixed \ indentation in long macros